File: src/main/java/ru/dragonestia/apocalypse/item/IronClusterItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class IronClusterItem extends Item implements AdvItem {

    public IronClusterItem(Integer meta, int count) {
        super(ApocalypseID.IRON_CLUSTER, meta, count, "Iron cluster");
    }

    @Override
    public String getRuName() {
        return "Железная руда";
    }

    @Override
    public String getIcon() {
        return "textures/items/cluster/iron";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/IronClusterItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class IronClusterItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.IRON_CLUSTER, 0, count, "Iron cluster"), AdvItem {

    override fun getRuName(): String = "Железная руда"

    override fun getIcon(): String = "textures/items/cluster/iron"

}